softlayer/disk/softlayer_creator.go

```diff
@@ -46,6 +47,8 @@ func (c SoftLayerCreator) Create(size int, cloudProps DiskCloudProperties, virtu
 		return SoftLayerDisk{}, bosherr.WrapError(err, "Create SoftLayer iSCSI disk error.")
 	}
 
+	time.Sleep(2 * time.Minute)
```

Comment:
Why are we sleeping here for 2 minutes straight. Can this be moved to some private method and best use some type of wait, check, and iterate until a max total wait... or until something is ready.

I don't understand what you are doing here